### PR TITLE
 tests: write cypress tests to check 0 day checkout

### PR DESCRIPTION
**Pull request by AoNoOokami**
* Adds a test to control that the 'zero day checkout' works correctly.
* Adds a test to check the creation of a circulation policy.

Co-Authored-by: Alicia Zangger <alicia.zangger@rero.ch>

## Why are you opening this PR?

To create cypress tests on cipo editor.

## Dependencies

My PR depends on the following `rero-ils-ui`'s PR(s):

* https://github.com/rero/rero-ils-ui/pull/469

## How to test?

- What command should I have to run to test your PR?
- What should I test through the UI?

## Code review check list

- [ ] Commit message template compliance.
- [ ] Commit message without typos.
- [ ] File names.
- [ ] Functions names.
- [ ] Functions docstrings.
- [ ] Unnecessary commited files?
- [ ] Cypress tests successful?

### CODE REVIEW

File: tests/e2e/cypress/cypress/integration/editor/ci-po/create-ci-po.spec.js [hunk 1]
```diff
@@ -0,0 +1,85 @@
+/// <reference types="Cypress" />
```
Comment (zannkukai) [inline tests/e2e/cypress/cypress/integration/editor/ci-po/create-ci-po.spec.js:1]: Is it normal the dash between `ci` and `po` into the name file ?

**Review by zannkukai — APPROVED** [review]

**Review by iGormilhit — CHANGES_REQUESTED** [review]
Commit message proposition:

```
tests: write cypress tests to check 0 day checkout

* Adds a test to control that the 'zero day checkout' works correctly.
* Adds a test to check the creation of a circulation policy.
```

File: tests/e2e/cypress/cypress/integration/editor/ci-po/create-ci-po.spec.js (hunk 1 above)
Comment (AoNoOokami) [reply tests/e2e/cypress/cypress/integration/editor/ci-po/create-ci-po.spec.js:1]: Yes

**Review by iGormilhit — APPROVED** [review]
Commit message approved.

**Review by benerken — APPROVED** [review]

**Pull request merged**